dotcom-rendering/src/server/lib/get-content-from-url.js

```diff
@@ -82,7 +82,7 @@ const parseURL = (requestURL, requestPath) => {
 	 */
 
 	let url = requestURL.includes('url=')
-		? requestURL.split('url=')[1]
+		? requestURL.split('url=')[1] ?? 'unknown'
```

Comment:
This could fail catastrophically, but it only happens on the development server.

Reply:
I have simplified this logic. If the url does not contain `url=`, then accessing the `[1]` will return undefined, and we default to the `requestUrl`. Simples.